### PR TITLE
add color warning if the user is debugging

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?

* Ticket: #
* Funded by: DFAT
* Description: 
  * add color warning if the user is debugging

debug

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by Gustry** [comment]
I can't get the sideci running anymore. I already forced push to retrigger the analysis, nothing more

**Comment by Gustry** [comment]
Ah ok, it finally came very late

File: safe/gui/widgets/dock.py [hunk 1]
```diff
@@ -611,6 +622,22 @@ def block_signals(self):
...
         self.hazard_layer_combo.blockSignals(True)
 
+    def debug_group_toggled(self):
+        """Helper to set a color on the debug button to know if it's debugging.
+
+        If debug analysis is true or if rounding is disabled, it will be
+        orange.
+        """
+        use_debug = self.use_debug_action.isChecked()
+        setting('use_debug_analysis', use_debug)
```
Comment (ismailsunni) [inline safe/gui/widgets/dock.py]: set expected type to `bool`

Comment (Gustry) [reply safe/gui/widgets/dock.py]: no, thanks it's a mistake. It must be 'set_setting'

Comment (ismailsunni) [reply safe/gui/widgets/dock.py]: ah yes, of course. That's why there is no assignment.

**Pull request merged**